api/spec/i18n_spec.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+require 'i18n/tasks'
+
+RSpec.describe I18n do
```

Comment:
Just a nit, but do you think it would make sense to turn this into a set of shared examples?

Bonus: would it make sense to add this to [solidus_dev_support](https://github.com/solidusio/solidus_dev_support)?

Reply:
> Just a nit, but do you think it would make sense to turn this into a set of shared examples?

As it is code automatically generated by the `i18n-tasks` gem, I think it's better to leave things as they're now. For instance, they could add a new task to update that code, which could not work in our case if we changed it.

> Bonus: would it make sense to add this to [solidus_dev_support](https://github.com/solidusio/solidus_dev_support)?

Yeah, that's an excellent point for the extensions.

Reply:
Hey @aldesantis, are you ok with that? :slightly_smiling_face:

Reply:
@aldesantis ping, so we can merge this one!

Reply:
Merging, we can always extract it as a follow-up work :slightly_smiling_face: